### PR TITLE
SVG

### PR DESCRIPTION
**Pull request by cmoreno-nts**
Check this out

### CODE REVIEW

File: reactjs/src/components/TodoContainer.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            if (todo.id === id) {
+              todo.title = updatedTitle
+            }
+            return todo
+          })
+        }) 
+        console.log(updatedTitle, id)
+        }
+        componentDidUpdate(prevProps, prevState) {
+  if(prevState.todos !== this.state.todos) {
```
Comment (jventura-nts) [inline reactjs/src/components/TodoContainer.js:59]: give a space between the if and "(" like if (prev...

File: reactjs/src/components/TodoItem.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import React from "react";
```
Comment (jventura-nts) [inline reactjs/src/components/TodoItem.js:1]: let's give formatting to this whole file

File: reactjs_ibaslogicrepo/src/FunctionBased/components/TodoContainer.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from "react";
+import Header from "./Header";
+import InputTodo from "./InputTodo";
+import TodosList from "./TodoList";
+import { v4  as uuidv4 } from "uuid";
+
+
+const TodoContainer = () => {
+  /*state = {
```
Comment (jventura-nts) [inline reactjs_ibaslogicrepo/src/FunctionBased/components/TodoContainer.js:9]: please remove commented code

**Comment by jventura-nts** [comment]
let's give formatting in general to the files, and also please resolve conflicts

**Review by adiaz-nts — APPROVED** [review]
Try to indent the code

**Pull request merged**